docs/ruleset_functions/get_primary_secondary_loops.md

```diff
@@ -0,0 +1,61 @@
+
+## get_primary_secondary_loops
+
+Description: Get the list of primary and secondary loops for HHW or CHW for a RMR.
+
+Inputs:  
+- **RMR**: The RMR that needs to get the list of primary and secondary loops.
+- **loop_type**: "COOLING" (or "HEATING", if needed in future)
+
+Returns: 
+- **primary_secondary_loop_dictionary**: A dictionary that saves the list of primary and secondary loops for the selected loop type, i.e. "HEATING" or "COOLING", e.g. {"PRIMARY": [loop_1.id, loop_2.id], "SECONDARY": [loop3.id]}. If primary loop or secondary loop is not modeled correctly in RMR, return an empty list for both.
+
```

Comment:
I find this function a bit confusing. Few thoughts:
1. I don't think producing a list of primary and secondary loops would work because it doesn't specify relationship. E.g., in your example above, it's not clear whether loop3 is associated with loop1 or loop2. 
2. For this function, why not just check the following:
a. Loops referenced by chillers are primary loops (LOOP List A)
b. Loops that are referenced by cooling systems (LOOP List B).
c. Baseline CHW primary/secondary is modeled correctly if the loop in List A (is NOT also in List B) AND (references the loop in List B as its child loop).

Reply:
Rule 22-8 might also need this. "For Baseline chilled water system with cooling capacity of 300 tons or more, the secondary pump shall be modeled with variable-speed drives."

Reply:
The current intent of the function is to return primary and secondary loops in a RMR, and not just judging if primary/secondary loops are modeled correctly. The reason for that is for rule 22-8, we will also need to know which loop is secondary to find the secondary pump. 
For Item#1 above, we can know the relationship by calling FluidLoop.child_loop for all primary loops. 
I think the current logic already checks 2a, 2b, 2c(ii) and 2c(iii), but it does not check 2c(i).